typescript/core/src/TempBuffer.ts

```diff
@@ -1,4 +1,4 @@
-import { IWritable } from "../common/IWritable";
+import { IWritable } from "./common/IWritable";
```

Comment:
I think the `common` directory can go away too. I believe it was meant to be common code between pre0 and v0, so the distinction is not necessary anymore.